src/App.tsx

```diff
@@ -55,6 +55,9 @@ const App: React.FC = () => {
             <Route path="/spells">
               <Farms tokenMode/>
             </Route>
+            <Route path="/referrals">
```

Comment:
This referral path is not required for the aurum front end. Referrals are only available in MIST.